src/FFmpegWriter.cpp

```diff
@@ -1100,6 +1121,16 @@ AVStream *FFmpegWriter::add_audio_stream() {
 
 	AV_COPY_PARAMS_FROM_CONTEXT(st, c);
 
+#if HAVE_CH_LAYOUT
+	ZmqLogger::Instance()->AppendDebugMethod(
```

Comment:
I would prefer we set a variable or two, instead of repeating the entire debug statement, just to keep things tidy.

Reply:
ok